tests/system/README.md

```diff
@@ -0,0 +1,85 @@
+# Layer0 System Tests
+
+### Test Architecture
+Each test case contains two components: a subdirectory under `cases/`, and a `<case name>_test.go` file.
+The test case directory contains terraform files that outline the resources needed for the test. 
+The testing framework will create these resources with `terraform apply` before executing the test, and delete them with `terraform destroy` once the test has completed.
+
+Note that the `cases/modules` directory is not a test case directory.
+It holds our [terraform modules](https://www.terraform.io/docs/modules/usage.html) to reduce duplicate code in our terraform files.
+
+If your test case was named `my_simple_example`, you should create the following files and directories:
+```
+layer0/tests/system
+|-- mySimpleExample_test.go
+|-- cases
+|   |-- my_simple_example
+|   |    |-- main.tf
+```
+
+### Writing a Test
+Writing a test is relatively straightforward.
+Following the example above, the contents of `mySimpleExample_test.go` should look like:
+```go
+package system
+
+import (
+        "testing"
+)
+
+// Test Resources:
+// This test creates an environment named 'sme' that has a
```

Comment:
where does the `sme` name get passed from? The `my_simple_example` file? If so, would be worth clarifying that.